public/leaderboardList.php

```diff
@@ -21,16 +21,18 @@
 
 $lbCount = getLeaderboardsList($consoleIDInput, $gameID, $sortBy, $count, $offset, $lbData);
 
+$pageTitle = "Leaderboard List";
```

Comment:
I'd go with "Leaderboards" like on game pages; "List" seems redundant?

Just noticed that it's "Achievements List" in `achievementList.php`; could be just "Achievements", too.

Reply:
That's unchanged (just move from line 48), but the change makes sense. I'll update it to match the breadcrumb since I changed that too.